assets/source/setup-guide/app/helpers/documentation-link-props.js

```diff
@@ -21,17 +33,18 @@ import { recordEvent } from '@woocommerce/tracks';
  *
  * Please be careful not to overwrite the `onClick` handler coincidently.
  *
- * @fires wcadmin_pfw_documentation_link_click on click, with given `linkId` and `context`.
+ * Notice documentation_link_click is the default eventName and pfw_ is not required
  *
+ * @fires wcadmin_pfw_documentation_link_click on click, with given `linkId` and `context`.
  * @param {Object} props React props.
  * @param {string} props.href Href to used by link and in track event.
- * @param {string} props.linkId Forwarded to {@link wcadmin_pfw_documentation_link_click}
- * @param {string} props.context Forwarded to {@link wcadmin_pfw_documentation_link_click}
+ * @param {string} props.linkId Forwarded to {@see wcadmin_pfw_documentation_link_click}
+ * @param {string} props.context Forwarded to {@see wcadmin_pfw_documentation_link_click}
```

Comment:
This is for preventing a few linter issues regarding the variable `wcadmin_pfw_documentation_link_click` the correct directive is @see I guess?

Reply:
in that usage, it's rather `{@link foo}`

Reply:
```
 * @see Bar used in a separate line is to link the entire block to Bar
 ```

Reply:
<img width="982" alt="Screenshot 2021-12-20 at 17 26 49" src="https://user-images.githubusercontent.com/5908855/146806065-26217e94-a4f9-44c8-a96d-d8a3462b83b4.png">
<img width="1307" alt="Screenshot 2021-12-20 at 18 21 50" src="https://user-images.githubusercontent.com/5908855/146807076-29d31a48-9c07-470d-a05e-be201a28e316.png">

If I rolled back to `@link` it gives me this warning. 

Rolled back here. https://github.com/woocommerce/pinterest-for-woocommerce/pull/299/commits/b419b62272a8aa688b4e219fe72c9ef649c14768

Reply:
Hmm.. that's strange `wcadmin_pfw_documentation_link_click` symbol is defined in the very same file. Maybe your IDE has problems with resolving the name path, or id does not support `@event` tag in JSDoc. Have you tried things like `event:wcadmin_pfw_documentation_link_click` or `#event:wcadmin_pfw_documentation_link_click`? 
What IDE do you use?